<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5SRPD8A" failures="1" tests="9" name="Car Loan" time="0.066" errors="0" timestamp="2022-04-06T20:18:50 SGT">
  <testcase classname="testng_tutorial.Day4" name="apiLoginHome" time="0.004"/>
  <testcase classname="testng_tutorial.Day4" name="mobileLoginHome" time="0.002"/>
  <testcase classname="testng_tutorial.Day4" name="mobileLoginHome" time="0.002"/>
  <testcase classname="testng_tutorial.Day4" name="mobileLoginHome" time="0.001"/>
  <testcase classname="testng_tutorial.Day4" name="mobileLoginHome2" time="0.001"/>
  <testcase classname="testng_tutorial.Day4" name="mobileLoginHome3" time="0.001"/>
  <testcase classname="testng_tutorial.Day4" name="sampleListener" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testng_tutorial.Day4.sampleListener(Day4.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- sampleListener -->
  <testcase classname="testng_tutorial.Day4" name="webLogin" time="0.001"/>
  <testcase classname="testng_tutorial.Day4" name="webLoginHome" time="0.001"/>
</testsuite> <!-- Car Loan -->
